### ISSUE TITLE
There are some deficiencies in automatic completion

### ISSUE THREAD

**Issue by ydmykr** [microsoft/pylance-release#3374]
fileA.py

```
import astropy.units as u

class A:
    def __init__(a: int = 1, b: u.Quantity[u.yr] = 0*u.yr) -> None:
        pass
```

fileB.py

```
import astropy.units as u

from fileA import A


class B(A):
```
If I type __init__ and use autocomplete. Pylance will give me the following code:

```
    def __init__(a: int = 1, b: u.Quantity[u.yr] = ...) -> None:
        super().__init__(b)
```
However, I want autocomplete can recognize `0*u.yr` in A instead of `...`

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in an upcoming version of pylance.

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2022.9.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022931-21-September-2022-PreRelease)
